addons/website/views/snippets/s_table_of_content.xml

```diff
@@ -5,10 +5,10 @@
     <section class="s_table_of_content pt24 pb24 o_cc o_cc1">
         <div class="container">
             <div class="row s_nb_column_fixed">
-                <div class="col-lg-3 s_table_of_content_navbar_wrap s_table_of_content_navbar_sticky s_table_of_content_vertical_navbar d-print-none d-none d-lg-block o_not_editable o_cc o_cc1" data-name="Navbar">
+                <div class="col-lg-3 s_table_of_content_navbar_wrap s_table_of_content_navbar_sticky s_table_of_content_vertical_navbar d-print-none d-none d-lg-block o_not_editable o_cc o_cc1 oe_unmovable" data-name="Navbar">
```

Comment:
This is actually an opportunity, as we are in Master here, to remove the .js code from "options.registry.TableOfContentMainColumns" which is used:
- To remove the "clone" button => this button will be removed by the "oe_unmovable" class now.
- To remove the "delete" button => this button is removed by the "oe_unremovable" class that we would add here and in the "s_table_of_content_main" element. 

=> Probably need an update script for the "table of contents" already dropped in the page.

Also, as this change impacts the UX, it would be nice to get @pierre-pde 's opinion on this.

Reply:
Created an upgrade script: https://github.com/odoo/upgrade/pull/4219.

Also removed the `TableOfContentMainColumns` JS class.

Reply:
> @bso-odoo I just saw this code too which could be removed I guess

This is about the table of content as a whole - which still needs to be prevented from being dropped inside another table of content.